<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchRender</class>
 <widget class="QWidget" name="BatchRender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Render</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="table">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tab_shots"/>
     <widget class="QPushButton" name="btn_up">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>˄</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_down">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>˅</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="stat">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="lab_frames_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>  Total Frames:  </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_totalFrames">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>  Frame Render Time (min):  </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_rt">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>15</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>  Total Render Time (days):  </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lin_rtTotal">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_addShots">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Add Shots</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delShots">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Del Shots</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reload">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_render">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Render</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
